theme/material/vaadin-confirm-dialog-styles.html

```diff
@@ -1,3 +1,14 @@
+<dom-module id="material-confirm-dialog-overlay" theme-for="vaadin-dialog-overlay">
```

Comment:
There's already a theme module for `vaadin-dialog-overlay` at the bottom of this file.

Reply:
Updated